core/sql/generator/GenExplain.cpp

```diff
@@ -754,16 +1017,30 @@ HbaseAccess::addSpecificExplainInfo(ExplainTupleMaster *explainTuple,
 
   }
 
+  // get column retrieved
+  if (((ComTdbHbaseAccess *)tdb)->listOfFetchedColNames()){
+	  description += "column_retrieved: ";
+	  appendListOfColumns(((ComTdbHbaseAccess *)tdb)->listOfFetchedColNames(),tdb,description);
+  }
+  // get predicate pushed down in Reverse Polish Notation for the AND / OR operators.
+  // could transform it standard notation for better readability, but good enough for now...
+  // could also evaluate the constants instead of hard coded ?, but good enough for now...
+  if (((ComTdbHbaseAccess *)tdb)->listOfHbaseFilterColNames()){
+  	  description += "pushed_down_rpn: ";
```

Comment:
How about I only change it if I see other changes that needs regenerating EXPECTED? If you feel strong, I'll change it, but letting you know that the impact is more than just one file... many EXPECTED test to generate and change? thought?